ovos_plugin_manager/templates/iot.py

```diff
@@ -427,7 +441,8 @@ def as_dict(self):
 
 
 DEVICE_TYPES = {
-    "generic": IOTDevicePlugin,
+    "generic_sensor": IOTSensorPlugin,
+    "generic_device": IOTDevicePlugin,
```

Comment:
a sensor reports state in some way, you may not necessarily be able to control it, you just know you receive data from it nothing else

eg, DIY iot device could emit a mqtt message, some custom sensor plugin keeps track of state

a device at a minimum has a on/off control, it is also a sensor because it tracks on/off state

Reply:
What about an object that can handle on/off commands but does not track state?

Reply:
whats an example of such device? I'm having a hard time coming up with a device that would do that

for sensors if you dont get a reading after X time you can assume it is offline, any reading means its online

for devices that handle on/off commands you know state whenever you issue a command, so you can also kinda track it internally after initial detection too

Reply:
If I wanted to hook up a garage door opener, there's only a single output to "toggle" the state. In practice, I'd probably pair this with a sensor to determine state of the door, but the "opener" object wouldn't have a state. 

For an IR controlled object, there wouldn't be a way to capture the state of the object, only send commands

Reply:
in the context of a voice framework the state is often implicit in the query, "turn on" implies state is off, the issue will be more in the GUI side, perhaps a unknown state representation makes more sense here ? the state exists in reality, we just dont have info about it